pkg/types/envd.go

```diff
@@ -240,12 +241,19 @@ func newDependencyFromLabels(label map[string]string) (*Dependency, error) {
 		}
```

Comment:
Could we add the unit test for the func?

Reply:
added